etc/mpsyt.profile

```diff
@@ -13,6 +13,18 @@ noblacklist ${HOME}/.netrc
 noblacklist ${HOME}/mps
 noblacklist ${MUSIC}
 noblacklist ${VIDEOS}
+noblacklist ${DOWNLOADS}
+
+mkdir ${HOME}/.config/mps-youtube
+
+whitelist ${HOME}/.config/mpv
+whitelist ${HOME}/.mplayer
+whitelist ${HOME}/.config/mps-youtube
+whitelist ${HOME}/.netrc
+whitelist ${HOME}/mps
+whitelist ${MUSIC}
+whitelist ${VIDEOS}
+whitelist ${DOWNLOADS}
 
```

Comment:
you should also add the following above the other whitelist inc
`include /etc/firejail/whitelist-common.inc`

Reply:
@SkewedZeppelin Shouldn't that be added without `/etc/firejail` as `include whitelist-common.inc`?